#
# old_revision []
#
# add_dir ""
# 
# add_file "a"
#  content [03cfd743661f07975fa2f1220c5194cbaff48451]
#
============================================================
--- /dev/null	
+++ a	03cfd743661f07975fa2f1220c5194cbaff48451
@@ -0,0 +1 @@
+abc
